scintillometry/tests/test_pintutils.py

```diff
@@ -0,0 +1,70 @@
+# Licensed under the GPLv3 - see LICENSE
+"""Full-package tests of pint_utils sources."""
+import pytest
+import numpy as np
+import os
+import astropy.units as u
+from astropy.time import Time
+pint_utils = pytest.importorskip('scintillometry.utils.pint_utils')
+
+
+test_data = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+
+
+class TestPintUtils:
+    """Test the utilities of PINT"""
+
+    def setup(self):
+        self.tim_file = os.path.join(test_data, 'NGC6440E.tim')
+        self.start_time = Time('2018-05-06T23:00:00', format='isot',
+                               scale='utc')
+        self.times = self.start_time + np.arange(1000) * 1.50 * u.min
+        self.obs = 'AO'
+        self.freq = 1400
+
+    def test_make_toalist(self):
+        toalist = pint_utils.make_toa_list(self.times, self.obs, self.freq)
+        assert len(toalist) == 1000, \
+            "TOA list did not made properly"
```

Comment:
error message not grammatical; I actually do not think it is necessary to have the messages for obvious errors, so would suggest just to remove it.